dmutils/logging.py

```diff
@@ -61,16 +60,12 @@ def get_handlers(app):
     standard_formatter = CustomLogFormatter(LOG_FORMAT, TIME_FORMAT)
     json_formatter = JSONFormatter(LOG_FORMAT, TIME_FORMAT)
 
-    # Log to files if the path is set, otherwise log to stderr
-    if app.config['DM_LOG_PATH']:
```

Comment:
Oh, I think we need to keep this for now since we still want to write two files in the existing EB environments. So before the switch we'll have DM_LOG_PATH set to empty in the PaaS environment while the EB environments use the path from the config files. 

After we shutdown EB apps we'll be able to remove this logic from utils and drop the config variable from application configs and PaaS environments.